en/docs/integrations/build-your-first-integration.md

```diff
@@ -1,59 +1,133 @@
+## Step 3: Get the population data
+Follow this procedure to connect to the world bank API and retrieve population data:
+
+1. Click the last **+** icon in the low-code diagram.
+2. Click **API Calls** and then click **World Bank API**.
+3. In the **World Bank API Connection** window, enter `worldBankClient` as the **Connection Name** and click **Save**.
+4. Click the last **+** icon in the low-code diagram and click **API Calls**.
+5. Under **Choose existing connection**, click **worldBankClient**.
+6. In the **Operation** drop-down list, select **Get Country Population** and enter details as follows in the other fields: 
+
+    1. In the **Country Code** field, enter `"USA"`.
+    2. In the **Date** field, enter `"2019"`.
+    3. Click **Optional**, and  enter `"json"` as the **Format**.
+    4. In the **Response Variable Name** field, enter `populationByCountry`.
+
+7. Click **Save**.
+8. Now let’s extract the population from the response and store it in a variable. Follow this procedure: 
+
+    1. Click the last **+** icon in the low-code diagram.
+    2. Under **Statements**, click **Variable** and enter details as follows:
+
+        | **Field**      | **Value**                     |
+        |----------------|-------------------------------|
+        | **Type**       | `int`                         |
+        | **Name**       | `population`                  |
+        | **Expression** | `(populationByCountry is worldbank:CountryPopulationArr ? populationByCountry[0]?.value ?: 0 : 0) / 1000000`       |
+
+    3. Click **Save**.
+
+## Step 4: Calculate the total COVID-19 case count by population 
+
+Now let’s calculate the total COVID-19 case count per million in the population based on the COVID-19 statistics and the population data you have retrieved. Follow this procedure:
+
+1. Click the last **+** icon in the low-code diagram.
+2. Under **Statements**, click **Variable** and enter details as follows:
+
+    | **Field**      | **Value**                     |
+    |----------------|-------------------------------|
+    | **Type**       | `var`                         |
+    | **Name**       | `totalCasesPerMillion`        |
+    | **Expression** | `totalCases / population`     |
+
+3. Click **Save**.
+         
+## Step 5: Generate and send an email with summary statistics
+Follow this procedure to send an email with the summary of statistics to a specified email address:
+
+1. Click the last **+** icon in the low-code diagram.
+2. Under **Statements**, click **Variable** and enter details as follows:
+
+    | **Field**      | **Value**                     |
+    |----------------|-------------------------------|
+    | **Type**       | `string`                      |
+    | **Name**       | `mailBody`                    |
+    | **Expression** | `"Per Million Data: Total Cases" + totalCasesPerMillion.toString()`|
```

Comment:
Expression is incorrect. Refer https://github.com/anupama-pathirage/covid/blob/master/integration.bal#L20

Reply:
Updated.